src/components/Input.vue

```diff
@@ -50,8 +64,10 @@ input[type="password"] {
   color: var(--text-input);
   height: 42px;
   margin-bottom: 24px;
-  max-width: 480px;
+  max-width: 320px;
```

Comment:
better create a variable in theme. 
Referencing the size. Usually 320px is small mobile-screen, you can reference as `screen.xs`